memorious/operations/aleph.py

```diff
@@ -132,6 +136,55 @@ def aleph_folder(context, data):
             backoff(ae, try_number)
 
 
+def aleph_emit_entity(context, data):
+    api = get_api(context)
+    if api is None:
+        return
+    collection_id = get_collection_id(context, api)
+    content_hash = data.get("content_hash")
+    source_url = data.get("source_url", data.get("url"))
+    foreign_id = data.get("foreign_id", data.get("request_id", source_url))
+    # Fetch id from cache
+    entity_id = context.get_tag(make_key(collection_id, foreign_id, content_hash))
```

Comment:
I still don't like the use of `content_hash` here. IMO this method is about all types of entities, not just articles.

Instead we should let the caller decide what keying strategy they want to use and expect them to pass in a unique key. For articles, that unique key can be created using content hash in the crawler's code.

Reply:
Yes, but we should still generate the namespaced caching key.